tools/cmake/README.md

```diff
@@ -19,12 +19,10 @@ The full profile with the selected printf and C libraries.
 Only a limited set of targets is supported at the moment.
 
 The following targets are supported:
-- DISCO_L475VG_IOT01A 
+- all STM32 targets 
```

Comment:
```suggestion
- All STM32 Mbed targets 
```

Reply:
Maybe not...?
We are working to ease the support for all custom boards with STM32. I hope that cmake feature keeps this goal.

Reply:
We do not count here custom targets. we ported all targets in the tree, or not?

Reply:
Yes, but STM32 targets that are not in mbed yet support cmake structure

Reply:
We will assist initially porting new targets, so any new PR labeled as "new target" will need to add CMake as well. I'll review opened PR and address this there. 

We are going to share the porting guide in the following days.